src/sql/workbench/browser/parts/views/treeView.ts

```diff
@@ -50,7 +50,7 @@ import { NodeContextKey } from 'sql/workbench/browser/parts/views/nodeContext';
 
 export class TreeViewPane extends ViewPane {
 
-	private treeView: ITreeView;
+	public treeView: ITreeView;
```

Comment:
I would recommend making a getter for these instead of making them public. Public fields in core logic can cause lots of problems.

Reply:
Yeah, if you don't need to allow callers to set it as well then using a getter only is preferable.

Reply:
I agree and that's how I defined it in the first place, but the getter method didn't return the treeView and was always null. So I had to stick to this instead. I'll be happy to show either of you what's happening and do the right thing here.

Reply:
Yeah I'm not clear what the problem would be. It would look something like this : 

```
	private _treeView: ITreeView;
	public get treeView(): ITreeView {
		return this._treeView;
	}
```

Reply:
Actually looking at it I think it's cleaner just to make it
`public readonly treeView: ITreeView`

since we're only setting it in the constructor.